src/main/io/osd.c

```diff
@@ -523,6 +523,13 @@ static bool osdDrawSingleElement(uint8_t item)
             tfp_sprintf(&buff[1], "%3d%c", h , SYM_DEGREES);
             break;
         }
+    case OSD_GPS_HDOP:
+        {
+            buff[0] = SYM_HDP_L;
+            buff[1] = SYM_HDP_R;
+            tfp_sprintf(&buff[2], "%2d", gpsSol.hdop / 100);
```

Comment:
@fiam  Can it use "HDOP_SCALE" in 'io/gps.h' ?
Like : tfp_sprintf(&buff[2], "%2d", gpsSol.hdop /  HDOP_SCALE  );

Reply:
@junwoo091400 Nice catch! It was an oversight on my part, since I was the one introducing the HDOP constant to get rid of that magic 100 and I ended up forgetting about it. Since this is already merged, would you mind submitting a PR with this change? Thanks!

Reply:
Sure!